test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -479,6 +517,128 @@ void ProtoConverter::visit(AssignmentStatement const& _x)
 	m_output << "\n";
 }
 
+template <class T>
+void ProtoConverter::visitFunctionInputParams(T const& _x, int _numInputParams)
+{
+	switch (_numInputParams)
+	{
+		case 0:
+			break;
+		case 1:
+			visit(_x.in_param1());
+			break;
+		case 2:
+			visit(_x.in_param1());
+			m_output << ", ";
+			visit(_x.in_param2());
+			break;
+		case 3:
+			visit(_x.in_param1());
+			m_output << ", ";
+			visit(_x.in_param2());
+			m_output << ", ";
+			visit(_x.in_param3());
+			break;
+		case 4:
+			visit(_x.in_param1());
+			m_output << ", ";
+			visit(_x.in_param2());
+			m_output << ", ";
+			visit(_x.in_param3());
+			m_output << ", ";
+			visit(_x.in_param4());
+			break;
+		default:
+			yulAssert(false, "Proto fuzzer: Function call with too many input parameters.");
+			break;
+	}
+}
+
+void ProtoConverter::visit(MultiAssignment const& _x)
+{
+	size_t funcId = (static_cast<size_t>(_x.func_index()) % m_functionVecMultiReturnValue.size());
+	int numInParams = m_functionVecMultiReturnValue.at(funcId).first;
+	int numOutParams = m_functionVecMultiReturnValue.at(funcId).second;
+	yulAssert(
+			((numOutParams >= 2) && (numOutParams <= 4)),
+			"Proto fuzzer: Multi assignment calls a function that has either too many or too few output parameters."
+	);
+
+	// Convert LHS of multi assignment
+	switch (numOutParams)
+	{
+		case 2:
```

Comment:
These can be combined if you don't use break (order would need to be reversed, but I guess that's fine):
```
case 4:
param4();
case 3:
param3();
...
```

Reply:
Ah, very cool :)

Reply:
I hope our compiler settings won't prevent that (the "missing" break might trigger a warning and our settings might consider that warning an error.). With C++17 fallthroughs could be explicitly marked :-): https://en.cppreference.com/w/cpp/language/attributes/fallthrough - we should probably move forward with that.

Reply:
done. Had to annotate with `BOOST_FALLTHROUGH;` otherwise clang complains of implicit fallthrough.